core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/arrow/ArrowNltkSentimentPredicate.java

```diff
@@ -0,0 +1,81 @@
+package edu.uci.ics.texera.dataflow.arrow;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+
+import edu.uci.ics.texera.api.exception.TexeraException;
+import edu.uci.ics.texera.dataflow.annotation.AdvancedOption;
+import edu.uci.ics.texera.dataflow.common.OperatorGroupConstants;
+import edu.uci.ics.texera.dataflow.common.PredicateBase;
+import edu.uci.ics.texera.dataflow.common.PropertyNameConstants;
+
+public class ArrowNltkSentimentPredicate extends PredicateBase {
+
+    private final String inputAttributeName;
+    private final String resultAttributeName;
+    private final String inputAttributeModel;
+    private final int batchSize;
+
+    @JsonCreator
+    public ArrowNltkSentimentPredicate(
+            @JsonProperty(value = PropertyNameConstants.ATTRIBUTE_NAME, required = true)
+                    String inputAttributeName,
+            @JsonProperty(value = PropertyNameConstants.RESULT_ATTRIBUTE_NAME, required = true)
+                    String resultAttributeName,
+
+            @AdvancedOption
+            @JsonProperty(value = PropertyNameConstants.NLP_NLTK_BATCH_SIZE, required = true,
```

Comment:
Since `ArrowNltkSentiment` is a new operator, it should not be using the PropertynameConstant `NLP_NLTK_BATCH_SIZE` which belongs to `NltkSentimentOperator`. You should add its own PropertynameConstant in `PropertyNameConsttants.java`. Something like `public static final String ARROW_NLTK_BATCH_SIZE = "batchSize";`